### DISCUSSION TITLE
Docker can't compose

### DISCUSSION THREAD

**Discussion by ineesdv** [TFNS/CTFNote#125] [Q&A]
I'm trying to host the CTFNotes site but I'm getting the following error:

`ubuntu@vm:/media/disk/home/CTF/CTFNote$ sudo docker-compose up `
Building front
Step 1/12 : FROM node:16-alpine AS build-deps
 ---> 8074a83d0ba9
Step 2/12 : RUN mkdir -p /usr/src/app
 ---> Using cache
 ---> aadcbb1f8dd3
Step 3/12 : COPY package.json quasar.conf.js .eslintrc.js .eslintignore tsconfig.json .postcssrc.js yarn.lock babel.config.js quasar.extensions.json /usr/src/app/
 ---> Using cache
 ---> fe130358b373
Step 4/12 : RUN cd /usr/src/app && yarn install
 ---> Running in f0a87fb1723b
yarn install v1.22.15
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
**error An unexpected error occurred: "https://registry.yarnpkg.com/hoist-non-react-statics/-/hoist-non-react-statics-3.3.2.tgz: getaddrinfo EAI_AGAIN registry.yarnpkg.com".**
info If you think this is a bug, please open a bug report with the information provided in "/usr/src/app/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
**ERROR: Service 'front' failed to build: The command '/bin/sh -c cd /usr/src/app && yarn install' returned a non-zero code: 1**

**Comment by XeR** [comment]
Hello,

It seems your virtual machine cannot resolve registry.yarnpkg.com.
Are you trying to install this on an air-grapped network ?
You need Internet during the installation process to fetch packages.

Maybe there is a problem with the configuration of your DNS server ?
Are you currently in a country where western websites are filtered ? (China, Iran, etc.)
